cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js

```diff
@@ -10,13 +11,28 @@ import SettingsMenu from '../../../support/fragments/settingsMenu';
 import TopMenu from '../../../support/fragments/topMenu';
 import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
 import InvoiceView from '../../../support/fragments/invoices/invoiceView';
-import DevTeams from '../../../support/dictionary/devTeams';
 
 describe('ui-data-import', () => {
-// unique name for profiles
-  const mappingProfileName = `autoTestMappingProf.${getRandomPostfix()}`;
-  const actionProfileName = `autoTestActionProf.${getRandomPostfix()}`;
-  const jobProfileName = `autoTestJobProf.${getRandomPostfix()}`;
+  const profileForDuplicate = FieldMappingProfiles.mappingProfileForDuplicate.harrassowitz;
+  const fileName = `C347615autotestFile.${getRandomPostfix()}.edi`;
+  const mappingProfile = {
+    name:`Import Large Harrassowitz Serials Invoice ${getRandomPostfix()}`,
+    incomingRecordType:NewFieldMappingProfile.incomingRecordType.edifact,
+    existingRecordType:NewFieldMappingProfile.folioRecordTypeValue.invoice,
+    description:'',
+    batchGroup: '"FOLIO"',
+    organizationName: NewFieldMappingProfile.organization.harrassowitz,
+    paymentMethod: '"Cash"'
+  };
+  const actionProfile = {
+    name: `Create Large Harrassowitz serials invoice ${getRandomPostfix()}`,
+    typeValue: 'Invoice',
```

Comment:
let's move this typeValue to the constants

Reply:
Done